sadx-dc-lighting/lantern.cpp

```diff
@@ -438,38 +438,37 @@ bool LanternInstance::load_palette(const std::string& path)
 
 	file.close();
 
-	if (color_data.size() > palette_pairs_.size())
+	unsigned int maxlength = palette_index_count * palette_index_length;
```

Comment:
Would prefer if this was named `max_length`. Type should be `size_t` to match the two operands. Also, since the two operands are `constexpr`, this could be made `constexpr` as well.